### DISCUSSION TITLE
Name Brainstorm

### DISCUSSION THREAD

**Discussion by nreinicke** [NREL/mappymatch#20] [General]
We just realized that there is already a `yamm` package on PyPI which could be confusing for users of this software (should have caught that sooner 😞) 

I'm starting this discussion as a brainstorm for other package names that we can adopt that won't conflict with existing PyPI packages.

Some initial ideas:
 - mappymatch
 - pymapmatch
 - pyyamm (probably too close to PyYAML)

**Comment by Rowlando13** [comment]
renewable energy map match (REMM). Open on pypi.

**Reply by Rowlando13** [reply]
Decision made. mappymatch (Map Py Match)
